### PR TITLE
Barebones of the jupiter contributing docs

### LINKED ISSUE #18: Add Contributing doc
# Overview

As a contributor, I should have a clear document describing the procedure to contribute to the repository.

# Proposal
Proposed process for making a change:

1. Ensure an issue is open in Github for that change
2. Create a new branch from develop, and name the new branch with the format `issue#-change-title`. An example is `3-org-card-spacing` which is addressing issue `3` which is about spacing of tiles in the directory page.
3. Once changes are completed on that branch, open a pull request to merge that into develop
4. Request review from engineering-reviewers (or other teams to be added later)
5. Iterate on the proposed change until at least one reviewer approves, at which point changes can be merged.

### PR DESCRIPTION
**Pull request by RubenOlano**
Closes #18 

Feel free to comment on anything else I should be adding to this, but this should cover at least the basics of what needs to be done to contribute to Jupiter

### CODE REVIEW

**Review by ZzRanger — COMMENTED** [review]
Looks solid! Just a few questions:
1. Are you giving recruits envs or are they setting them up themselves? Based on readme it seems like the former, but I'm wondering if it would be worthwhile to have them do the latter?
2. Are you using GitHub or JIRA for issue tracking? Might be helpful to include a blurb on how to find a ticket to work on?
3. `how to access the Jupiter Dev Team` is a bit vague. Maybe you can say "Join Discord & drop a message in the jupiter channel OR DM Ruben"

File: README.md [hunk 1]
```diff
@@ -2,49 +2,42 @@
...
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deployment
+```bash
+git checkout -b feature/<feature-name>
+```
 
-The `develop` branch for this repository is automatically deployed to Vercel at [dev.jupiter.utdnebula.com](https://dev.jupiter.utdnebula.com).
+When you are ready to merge your branch into the `develop` branch, please create a pull request and request a review from the Jupiter Dev Team
```
Comment (Master7130) [inline README.md:43]: Maybe add more details on what should be included in the pull request (issue number, brief description of the changes, etc)

File: README.md [hunk 2]
```diff
@@ -2,49 +2,42 @@
...
 _A tool to find ways to get involved on campus._
 
-## Getting Started
+## Contributing
 
-This project requires a working [Node.js](https://nodejs.org/en/) installation
-(at least v14) with NPM, which comes bundled with Node.js (at least v8). (If you
-use Yarn, that's fine, too. Just beware of the commands.)
+We are always open to contributions to the project. If you would like to contribute,
+please join our [Discord](http://discord.utdnebula.com/) and ask for how to access the Jupiter Dev Team
```
Comment (Jake3231) [inline README.md:8]: Does this imply that the project isn't open for public OSS contributions?

File: README.md [hunk 3]
```diff
@@ -2,49 +2,42 @@
...
 
-First, clone the repository.
+Please make sure you have at least NodeJS v18.0.0 or greater installed before continuing.
```
Comment (Jake3231) [inline README.md:10]: It might be worth linking to the standard setup/install docs for Node here

**Comment by RubenOlano** [comment]
I'll look into adding some more documentation for adding your own env variable

> Looks solid! Just a few questions:
> 
> 1. Are you giving recruits envs or are they setting them up themselves? Based on readme it seems like the former, but I'm wondering if it would be worthwhile to have them do the latter?
> 2. Are you using GitHub or JIRA for issue tracking? Might be helpful to include a blurb on how to find a ticket to work on?
> 3. `how to access the Jupiter Dev Team` is a bit vague. Maybe you can say "Join Discord & drop a message in the jupiter channel OR DM Ruben"

Worked on some rewording for this. Now that we've moved away from the firebase stuff it is a lot more feasible to set up your own environment so I'll look into adding a section for this

File: README.md (hunk 2 above)
Comment (RubenOlano) [reply README.md]: I think if I add a section about setting up your own environment variables then I can reword this line a bit more. I'll work on it and then resolve this issue once I'm done with that

File: README.md (hunk 3 above)
Comment (RubenOlano) [reply README.md]: Added link 👍🏽

File: README.md (hunk 2 above)
Comment (Jake3231) [reply README.md]: Looks like overall it is improved 👍

**Review by ZzRanger — APPROVED** [review]
LGTM!

**Pull request merged**